lab/13-npm/package.json

```diff
@@ -1,17 +1,24 @@
 {
+  "dependencies": {
```

Comment:
These are actually `devDependencies` as you use them in tests, not in the source code of your package.